doc/command_guidelines.md

```diff
@@ -281,13 +281,9 @@ class FooRuleAddAction(argparse._AppendAction):
     def __call__(self, parser, namespace, values, option_string=None):
         try:
             FooRule = namespace._cmd.get_models('FooRule')
-            name, metric, operation, value = values.split()
-            return FooRule(
-                name=name,
-                metric=metric,
-                operation=operation,
-                value=value
-            )
+            name, metric, operation, value = values
+            rule = FooRule(name=name, metric=metric, operation=operation, value=value)
+            super(FooRuleAddAction, self).__call__(parser, namespace, rule, option_string)
```

Comment:
Maybe we can split the changes on docs to another PR? Seems they have no relationship with the new commands.

Reply:
Sure, deleted from this PR. I'll open another.